controller/backup_volume_controller.go

```diff
@@ -284,11 +284,26 @@ func (bvc *BackupVolumeController) reconcile(backupVolumeName string) (err error
 		return err
 	}
 
+	// Get `Time to live` after failed backup was marked as `Error` or `Unknown`,
+	// default: 180s and it is shorter than `Backupstore Poll Interval` by default, so failed backups should be deleted once the backup volume is updated.
```

Comment:
The default was changed not 180 seconds. This comment is not needed anymore.

Reply:
Removed. Thanks.